input/haleu/sensitivity-analysis/global/mmr_share/mmr_share_input.xml.in

```diff
@@ -0,0 +1,248 @@
+  <xi:include href="../sensitivity-analysis/global/mmr_share/cyclus-files/xe100_{{xe100_burnup}}.xml"/>
+  <xi:include href="../sensitivity-analysis/global/mmr_share/cyclus-files/mmr_{{mmr_burnup}}.xml"/>
+  <xi:include href="united_states/reactors/nuscale_voygr.xml"/>
+  <facility>
+    <name>Sink_HLW</name>
+    <config>
+      <Sink>
+        <in_commods>
+          <val>casked_spent_uox</val>
+          <val>cooled_spent_xe100_haleu</val>
+          <val>cooled_spent_mmr_haleu</val>
+          <val>cooled_spent_smr_fuel</val>
+        </in_commods>
+        <!-- Yucca Mountain Repository -->
+        <latitude>36.8286</latitude>
+        <longitude>-116.4364</longitude>
+      </Sink>
+    </config>
+  </facility>
+  <facility>
+    <name>Sink_LLW</name>
+    <config>
+      <Sink>
+        <in_commods>
+          <val>tails</val>
+        </in_commods>
+        <!-- Yucca Mountain Repository -->
+        <latitude>36.8286</latitude>
+        <longitude>-116.4364</longitude>
+      </Sink>
+    </config>
+  </facility>
+  <facility>
+    <name>Mine</name>
+    <config>
+      <Source>
+        <outcommod>nat_u</outcommod>
+        <outrecipe>nat_u</outrecipe>
+        <throughput>1e6</throughput>
+        <!-- Cameco Smith Ranch-Highland Mine -->
+        <latitude>43.053736</latitude>
+        <longitude>-105.685089</longitude>
+      </Source>
+    </config>
+  </facility>
+  <facility>
+    <name>Enrichment</name>
+    <config>
+      <Enrichment>
+        <feed_commod>nat_u</feed_commod>
+        <feed_recipe>nat_u</feed_recipe>
+        <product_commod>fresh_uox</product_commod>
+        <tails_commod>tails</tails_commod>
+        <tails_assay>0.002</tails_assay>
+        <max_enrich>0.16</max_enrich>
+        <!-- Portsmouth Gaseous Diffusion Plant (OH) -->
+        <latitude>39.008333</latitude>
+        <longitude>-83</longitude>
+      </Enrichment>
+    </config>
+  </facility>
+  <facility>
+    <name>WetStorage</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>spent_uox</val>
+        </in_commods>
+        <in_recipe>spent_uox</in_recipe>
+        <out_commods>
+          <val>cool_spent_uox</val>
+        </out_commods>
+        <residence_time>72</residence_time>
+        <!-- Yucca Mountain Repository -->
+        <latitude>36.8286</latitude>
+        <longitude>-116.4364</longitude>
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>DryStorage</name>
+    <config>
+      <Storage>
+        <in_commods>
+          <val>cool_spent_uox</val>
+        </in_commods>
+        <in_recipe>cool_spent_uox</in_recipe>
+        <out_commods>
+          <val>casked_spent_uox</val>
+        </out_commods>
+        <!-- Yucca Mountain Repository -->
+        <latitude>36.8286</latitude>
+        <longitude>-116.4364</longitude>
+      </Storage>
+    </config>
+  </facility>
+  <facility>
+    <name>Xe-100CoolingPool</name>
```

Comment:
(A half baked question about the location of the cooling pools): Is this saying that you're deploying an MMR, VOYGR, and xe100 at Yucca mountain? Or is that not what these pools are?